include/zephyr/bluetooth/gatt.h

```diff
@@ -296,8 +296,14 @@ struct bt_gatt_attr {
 	 *  satisfied before calling read() or write().
 	 *
 	 *  @sa bt_gatt_discover_func_t about this field.
+	 *
+	 *  @note The `_auto_assigned_handle` flag in this bitfield indicates if the attribute
+	 *  handle was automatically assigned by the stack (1) or manually set by the
+	 *  application (0).
+	 *  This flag must not be modified by the application.
```

Comment:
This should be moved to a separate Doxygen comment for `_auto_assigned_handle`, and should include `@internal`.

Reply:
`@internal` cannot be used to hide struct members from documentation. `@cond INTERNAL_HIDDEN` needs to be used for that purpose

Reply:
I played around with this. It seems Doxygen will hide members that being with an underscore. It's the same effect as adding `@private`.

@akredalen, Since this field starts with an underscore, it's already private and there is no need to add anything. Sorry for the confusion.

Reply:
@kartben can you confirm?

Reply:
Seems correct, e.g. https://docs.zephyrproject.org/latest/doxygen/html/structbt__conn__cb.html does not show 

```
	/** @internal Internally used field for list handling */
	sys_snode_t _node;
```

Reply:
So, remove current lines 302 and 311 then? @alwa-nordic All other text is ok as is?